### ISSUE TITLE
Batch_Normalization appears to cause issues

### ISSUE THREAD

**Issue by monishramadoss** [Microsoft/Windows-Machine-Learning#25]
**Currently I am working on a super resolution project and I am Unable to load my model using winml_prerelease version. After disabling batchnormalization it works which is strange because the mdoel works in cntk. Below I have put my error.**

System.Runtime.InteropServices.COMException
  HResult=0x80004005
  Message=Unspecified error

Required attribute 'consumed_inputs' is missing.
  Source=System.Private.CoreLib
  StackTrace:
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at SuperResolutionProject_Infer.Model.<CreateFromStorageFile>d__3.MoveNext() in C:\Users\mramados\source\repos\App\SuperResolutionProject_Infer\4xmodel.cs:line 29
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at SuperResolutionProject_Infer.NeuralNetworkTask.<Eval_NN>d__24.MoveNext() in \NeuralNetworkTask.xaml.cs:line 219

**Comment by walrusmcd** [comment]
Hello !   Do you have some more details around your model and how you are loading it in winml ?

Here is the link to the [ONNX BatchNormalization operator ](https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization ) .

How did you create your model?     Are you using CNTK to produce the ONNX model?    Windows ML requires ONNX version 1.2.2 , which you need to make sure you are getting from CNTK.

We can help, post a few more details and we will get you going.  thanks !

**Comment by monishramadoss** [comment]
I am loading the model through uwp app I have developed although I have tried to load the neural net using squeezeNet example and I get the same error. I created the neural network with CNTK 2.5.1 (latest) and the model works in CNTK as I have tested it extensively to see if it is an issue with onnx but I have run into issues.  My windows build is Win10 enterprise 17733.rs5_release using the 17733 sdk.
